gaming-website/src/javaScript/games.json

```diff
@@ -0,0 +1,121 @@
+{
+  "payload": {
+    "gamesCover": [
```

Comment:
This key doesn't make a lot of sense in name with the new structure

Reply:
that's true